.vib/keycloak/goss/vars.yaml

```diff
@@ -0,0 +1,7 @@
+binaries:
+  - java
+  - wait-for-port
+directories:
+  - paths:
+      - /opt/bitnami/keycloak/modules/system/layers/keycloak/org/postgresql/jdbc/main
```

Comment:
Given that you are already checking the `module.xml` file inside this dir, I think we can remove this check.